.circleci/config.yml

```diff
@@ -993,7 +1032,7 @@ jobs:
           name: "yarn-project"
           command: yarn-project/deploy_npm.sh
 
-  deploy-dockerhub:
+  deploy-dockerhub-sandbox:
```

Comment:
If you have a look at the deploy_npm step, it recently changed to call a single script in yarn-project which deploys all of the relevant packages.. Could we make the deploy_dockerhub step similar to that?